server/models/Collective.js

```diff
@@ -868,6 +868,23 @@ export default function(Sequelize, DataTypes) {
     }
   };
 
+  /**
+   * If the collective is a host, it needs to remove existing hosted collectives before
+   * deactivating it as a host.
+   */
+  Collective.prototype.deactivateAsHost = async function() {
+    const hostedCollectivesCount = await this.getHostedCollectivesCount();
+    if (hostedCollectivesCount >= 1) {
+      throw new Error(
+        `You cannot deactivate your collective while still hosting ${hostedCollectivesCount} other collectives. Please contact support.`,
```

Comment:
The copy is unclear, it's not deactivating the collective.

Please contact support: support@opencollective.com.